### PR TITLE
feat: implement color normalization

### PR DESCRIPTION
**Pull request by wildergd**
fix: #1327

## Changes proposed in this PR:
- add colorToRgba helper
- add normalizeThemeColors helper

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/styles/helpers/color/colorToRgba.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if (typeof color !== 'string') return '';
+
+    try {
+        const { type, values } = decomposeColor(color);
+
+        if (!['rgb', 'rgba', 'hsl', 'hsla'].includes(type)) return '';
+
+        if (type === 'rgb') return rgbToRgba(color);
+        if (type === 'hsl') return rgbToRgba(hslToRgb(color));
+        if (type === 'hsla') return hslToRgb(color);
```
Comment (LeandroTorresSicilia) [inline src/styles/helpers/color/colorToRgba.js]: in this case we will return rgb instead of rgba?

File: src/styles/helpers/color/colorToRgba.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+        return recomposeColor({ type, values });
```
Comment (LeandroTorresSicilia) [inline src/styles/helpers/color/colorToRgba.js]: I assume that this is the rgba case, is not better to return the color directly without need to pass through recomposeColor?

File: src/styles/helpers/normalizeThemeColors.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            if (typeof value === 'string') {
+                normalizedPalette[key] = colorToRgba(value);
+            } else if (typeof value === 'object') {
+                normalizedPalette[key] = normalizePaletteColors(palette[key]);
+            } else {
+                normalizedPalette[key] = palette[key];
+            }
+        });
+        return normalizedPalette;
+    };
```
Comment (LeandroTorresSicilia) [inline src/styles/helpers/normalizeThemeColors.js]: let's put this function outside normalizeThemeColors function but in this same file

File: src/styles/helpers/color/colorToRgba.js (hunk 1 above)
Comment (wildergd) [reply src/styles/helpers/color/colorToRgba.js]: no the hslToRgb helper returns an rgba if hsla colors is passed as param

**Pull request merged**